cli/asc.js

```diff
@@ -501,6 +501,9 @@ exports.main = function main(argv, options, callback) {
 
   // Set up transforms
   const transforms = [];
+  if (Array.isArray(options.transforms)) {
+    transforms.push(...options.transforms);
+  }
```

Comment:
```suggestion
  const transforms = options.transforms || [];
```

Reply:
Mostly trying to be careful here by not modifying what's passed in, so subsequent calls with the same `options.transforms` value start out fresh again instead of inheriting potential modifications done by `asc.main`.